src/main/target/MATEKF411RX/target.h

```diff
@@ -42,6 +42,9 @@
 
 #define USE_SPI
 
+#undef  ENABLE_DSHOT_DMAR
```

Comment:
Can you please remove this - this will probably break all kinds of configurations that work now.